src/ExportMap.js

```diff
@@ -445,6 +449,21 @@ ExportMap.parse = function (path, content, context) {
 
   const source = makeSourceCode(content, ast)
 
+  function isEsModuleInterop() {
+    const tsConfig = tsConfigLoader({
+      cwd: context.parserOptions && context.parserOptions.tsconfigRootDir || process.cwd(),
+      getEnv: (key) => process.env[key],
+    })
+    try {
+      if (tsConfig.tsConfigPath !== undefined) {
+        const json = fs.readFileSync(tsConfig.tsConfigPath)
+        return JSON.parse(json).compilerOptions.esModuleInterop
```

Comment:
i don't believe that tsconfig is valid JSON, since it allows comments. I think we have to use whetever parsing package typescript itself uses to be able to parse this?

Reply:
Yep, valid point, they use their own parser, so I added commit to address this